apps/web/pages/availability/index.tsx

```diff
@@ -99,13 +99,14 @@ export function AvailabilityList({ schedules }: inferQueryOutput<"viewer.availab
   );
 }
 
+const WithQuery = withQuery(["viewer.availability.list"]);
+
 export default function AvailabilityPage() {
   const { t } = useLocale();
-  const query = trpc.useQuery(["viewer.availability.list"]);
   return (
     <div>
       <Shell heading={t("availability")} subtitle={t("configure_availability")} CTA={<NewScheduleButton />}>
-        <QueryCell query={query} success={({ data }) => <AvailabilityList {...data} />} />
+        <WithQuery success={({ data }) => <AvailabilityList {...data} />} />
```

Comment:
Just a low hanging fruit similar to `event-types` and `profile` page to apply the optimization using `withQuery`